datamonster_api/lib/aggregation.py

```diff
@@ -7,7 +7,15 @@
 aggregation_periods = {"week", "month", "quarter", "fiscalQuarter", "year"}
 
 Aggregation = namedtuple("Aggregation", ["period", "company"])
+"""
+A representation of an aggregation type within Data Monster
 
+:param str period: The period of time over which to aggregate. Valid choices are: 
+    `week`, `month`, `quarter`, `fiscalQuarter`, or `year`.
```

Comment:
you can hook in `aggregation_periods` instead of duplicating (what happens when it changes, need to update twice?)